src/main/webapp/app/view/project/admin/project-admin.module.ts

```diff
@@ -6,19 +6,26 @@ import { ProjectAdminComponent } from 'app/view/project/admin/project-admin.comp
 import { ProjectNavbarComponent } from 'app/view/project/admin/navbar/project-navbar.component';
 
 import { ProjectLocationsComponent } from 'app/view/project/admin/locations/project-locations.component';
+import { ProjectLocationUpdateComponent } from 'app/view/project/admin/locations/project-location-update.component';
 import { ProjectResponsibilitiesComponent } from 'app/view/project/admin/responsibilities/project-responsibilities.component';
 import { ProjectResponsibilityUpdateComponent } from 'app/view/project/admin/responsibilities/project-responsibility-update.component';
 import { ProjectResponsibilityDeleteDialogComponent } from 'app/view/project/admin/responsibilities/project-responsibility-delete-dialog.component';
 import { ProjectUsersComponent } from 'app/view/project/admin/users/project-users.component';
 
 import { PROJECT_ADMIN_ROUTES } from './project-admin.routes';
+import { ProjectLocationModalComponent } from 'app/view/project/admin/locations/project-location-modal.component';
+import { ProjectLocationDeleteDialogComponent } from 'app/view/project/admin/locations/project-location-delete-dialog.component';
+import { NgxAutocompleteModule } from '@bbortt/ngx-autocomplete';
```

Comment:
move the module import to the others, components too. constant `import { PROJECT_ADMIN_ROUTES } from './project-admin.routes';` is the last one.

Reply:
ah wait, actually the `NgxAutocompleteModule` shouldn't be required here. you already have it in `view.module`! please remove it.

Reply:
Wenn ich es entferne, dann erhalte ich die Fehlermeldung "Property options is not provided by any applicable directives nor by ngx-autocomplete element "

Reply:
achso, mein fehler. der component ist ja hier deklariert. dann im view-module entfernen.